src/openloco/input/mouse_input.cpp

```diff
@@ -1406,18 +1406,13 @@ namespace openloco::input
         // Not implemented for any window
         // window->call_22()
 
-        registers regs;
-        regs.eax = outX;
-        regs.ebx = outY;
-        regs.cx = (int16_t)scrollArea;
-        regs.edx = scrollAreaOffset;
+        ui::window* w = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
 
         switch (scrollArea)
         {
             case ui::scrollview::scroll_part::hscrollbar_button_left:
             {
                 // 0x004C894F
-                ui::window* w = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
                 if (w != nullptr)
```

Comment:
Can you also move the `if (w == nullptr) ` to before the switch.

Reply:
Sure, no problem.